README.md

```diff
@@ -158,6 +158,12 @@ Log all metrics, traces, and logs that are created for debugging purposes (in ad
 
 This will also send metrics and traces to Loki as an unintended side effect.
 
+#### grafana.otlp.enabled
+
```

Comment:
Should we mention that the default is enabled (i.e. true)?

Reply:
yes, good point :smile: 

Did you notice that this entire comment was generated by copilot? :wink: